lightning/src/onion_message/packet.rs

```diff
@@ -245,7 +257,7 @@ impl<H: CustomOnionMessageHandler> ReadableArgs<(SharedSecret, &H)> for Payload<
 				Ok(Payload::Receive {
 					control_tlvs: ReceiveControlTlvs::Unblinded(tlvs),
 					reply_path,
-					message: OnionMessageContents::Custom(message.unwrap()),
+					message: message.unwrap(),
```

Comment:
nit, and not related to changed line, but probably nice to avoid the (benign) unwrap in this case:

```rust
if let Some(message) = message {
    Ok(Payload::Receive {
        control_tlvs: ReceiveControlTlvs::Unblinded(tlvs),
        reply_path,
        message,
    })
} else {
    Err(DecodeError::InvalidValue)
}

```